src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperConventions.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Razor.TagHelpers
+{
+    /// <summary>
+    /// Default convention for determining if a type is a tag helper.
+    /// </summary>
+    public static class TagHelperConventions
```

Comment:
Is this just for consistency with the `ViewComponent` system or is there another need for this?